src/WinGetUtil/WinGetUtil.h

```diff
@@ -150,6 +156,33 @@ extern "C"
         WINGET_STRING mergedManifestPath,
         WinGetValidateManifestOption option);
 
+    // Creates a given manifest with optional validation. Returns a bool for operation result and
+    // a string representing validation errors if validation failed.
+    // If mergedManifestPath is provided, this method will write a merged manifest
+    // to the location specified by mergedManifestPath
+    WINGET_UTIL_API WinGetCreateManifest(
+        WINGET_STRING inputPath,
+        BOOL* succeeded,
+        WINGET_SQLITE_MANIFEST_HANDLE* manifest,
+        WINGET_STRING_OUT* message,
+        WINGET_STRING mergedManifestPath,
+        WinGetCreateManifestOption option);
+
+    // Validates a given manifest. Returns a bool for validation result and
+    // a string representing validation errors if validation failed.
+    // If mergedManifestPath is provided, this method will write a merged manifest
+    // to the location specified by mergedManifestPath
+    WINGET_UTIL_API WinGetValidateManifestV2(

```

Comment:
```suggestion
    WINGET_UTIL_API WinGetValidateManifestV3(
```
And change the parameters to match.